composer.json

```diff
@@ -44,6 +44,7 @@
             "drupal/console-extend-plugin": true,
             "drupal/core-composer-scaffold": true,
             "zaporylie/composer-drupal-optimizations": true
```

Comment:
You are missing a comma,
Give this a `composer install` in the future before submitting the pr when you make a change. 

Check out the feedback on the CI also if you get the chance.
https://app.circleci.com/pipelines/github/fourkitchens/sous-drupal-project/239/workflows/2f124e0b-a18d-4705-8dfa-2b3d5009e2de/jobs/407?invite=true#step-103-4